generic_config_updater/services_validator.py

```diff
@@ -48,6 +48,18 @@ def _service_restart(svc_name):
     return rc == 0
 
 
+def rsyslog_validator(old_config, upd_config, keys):
+    old_syslog = old_config.get("SYSLOG_SERVER", {})
+    upd_syslog = upd_config.get("SYSLOG_SERVER", {})
+
+    if old_syslog != upd_syslog:
+        os.system("systemctl reset-failed rsyslog-config rsyslog")
```

Comment:
>system [](http://example.com/codeflow?start=11&length=6)

@maipbui to review: why this line does not fail semgrep checker?

Reply:
it's static value, not dynamic value so the checker will not fail

Reply:
However, we can consider use `subprocess.call` here as it's better option, ref: https://stackoverflow.com/questions/44730935/advantages-of-subprocess-over-os-system, https://docs.python.org/3/library/os.html#os.system